test/helper.rb

```diff
@@ -23,7 +23,7 @@ module ARHPTestSetup
 
   def arhp_create_models
     return if Object.const_defined?('Test1')
-    eval <<-EOL
+    eval <<-CREATE_MODELS
```

Comment:
use `<<-RUBY` to get syntax highlighting

Reply:
I'm pretty sure that Rubocop won't like that

Reply:
totally works